docs/stream-aggregation.md

```diff
@@ -334,35 +336,190 @@ For example, the following config removes the `:1m_sum_samples` suffix added [to
 
 ## Aggregation outputs
 
-The following aggregation outputs can be put in the `outputs` list at [stream aggregation config](#stream-aggregation-config):
+The aggregations are calculated during the `interval` specified in the [config](#stream-aggregation-config)
+and then sent to the storage.
+
+If `by` and `without` lists are specified in the [config](#stream-aggregation-config),
+then the [aggregation by labels](#aggregating-by-labels) is performed additionally to aggregation by `interval`.
+
+Below are aggregation outputs that can be put in the `outputs` list at [stream aggregation config](#stream-aggregation-config).
+
+### total
 
 * `total` generates output [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) by summing the input counters.
-  The `total` handler properly handles input counter resets.
-  The `total` handler returns garbage when something other than [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) is passed to the input.
+  * The `total` handler properly handles input counter resets.
+  * The `total` handler returns garbage when something other than [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) is passed to the input.
```

Comment:
handler returns garbage -> we need to change wording for that

Reply:
I didn't get the meaning of the sentence. It should be re-phrased in a more simple manner

Reply:
@Amper prefer simple and straight wording. This makes reading easier for users.
See https://developers.google.com/tech-writing

Reply:
Ok, i'll try to reword it, but this particular text was written before my changes :)

Reply:
fixed